.eslintrc.js

```diff
@@ -188,6 +190,7 @@ module.exports = {
             '@typescript-eslint/no-unused-vars': 'off',
             '@typescript-eslint/no-unused-vars-experimental': 'off',
             '@typescript-eslint/dot-notation': 'off',
+            'github/array-foreach': 'off',
```

Comment:
I disabled the rule for unit tests, since micro-optimizing the tests doesn't really matter...